app/ents/employee/endpoints.py

```diff
@@ -77,25 +84,40 @@ def login_employee():
 
     employee = authenticate(crud, email, password)
     if employee:
-        return success_response(
+        access_token = create_access_token(employee.id, expires_delta=timedelta(minutes=settings.ACCESS_TOKEN_EXPIRE_MINUTES))
+        refresh_token = create_refresh_token(employee.id, expires_delta=timedelta(minutes=settings.REFRESH_TOKEN_EXPIRE_MINUTES))
+
+        resp= success_response(
```

Comment:
How about setting the response cookie online 90 and then returning success_response after that? Just to simplify the code

Reply:
Review and merge my current PR.
success_response now returns a Flask Response object. You can use the set_cookie method on that returned object to set the refresh token in the cookie

Reply:
okay great!